install.sh

```diff
@@ -2,41 +2,49 @@
 cd `dirname $0`
 set -e
 
+printf "Looking for javac... "
 if [ ! -f /usr/bin/javac ] ; then
-    # Need to update or else the installs won't work
+    printf "... not found. Installing packages... "
```

Comment:
In my expereince, it's more idiomatic to do `echo -n` in a shell script, though this is OK too.